docker-compose.yml

```diff
@@ -75,7 +75,7 @@ services:
       retries: 3
       start_period: 40s
     hostname: frontend
-    image: node:14
+    image: node:16.14.0-alpine
```

Comment:
I think @mishraomp was working on updating our docker images and we were updating to node 22. It might be a good idea to sync up with him. I assume that if this value gets updated later it won't hurt anything.